central/version/postgres/store.go

```diff
@@ -49,11 +49,13 @@ func insertIntoVersions(ctx context.Context, tx *postgres.Tx, obj *storage.Versi
 	}
 
 	values := []interface{}{
-		// parent primary keys start
+		obj.GetSeqNum(),
+		obj.GetVersion(),
+		obj.GetMinSeqNum(),
 		serialized,
 	}
 
-	finalStr := "INSERT INTO versions (serialized) VALUES($1)"
+	finalStr := "INSERT INTO versions (seqnum, version, minseqnum, serialized) VALUES($1, $2, $3, $4)"
```

Comment:
Maybe not for this PR but can we just remove serialized? If serialized is used then we could never do any manual magic that we may want to, but if it is not, then we could perform surgery

Reply:
@connorgorman I think that makes sense.  probably need to think through when the right time would be.

Reply:
That also requires to make the schema bespoke or figure out how to flag the generator to not build `serialized`